src/index.js

```diff
@@ -303,14 +303,48 @@ var reactGA = {
      * require:
      * GA requires a plugin
      * @param name {String} e.g. 'ecommerce' or 'myplugin'
+     * @param options {Object} optional e.g {path: '/log', debug: true}
      */
-    require: function (name) {
+    require: function (name, options) {
       if (typeof ga === 'function') {
-        ga('require', name);
-      }
 
-      if (_debug) {
-        log('called ga(\'require\', \'' + name + '\');');
+        // Required Fields
+        if (!name) {
+          warn('name is required in .require()');
+          return;
+        }
+
+        name = trim(name);
+        if (path === '') {
```

Comment:
Tests not passing because of this error. Path does not exist. Please fix this and I think we are done (also fix the warn message.)